project_fields_sequence/models/project_project.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2023 Moduon Team S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3.0)
+
+
+from odoo import api, models
+
+PROJECT_FIELDS_SEQUENCE_CODE_BASE = "project.field."
+
+
+class ProjectProject(models.Model):
+    _name = "project.project"
+    _inherit = "project.project"
+
+    @api.model
+    def default_get(self, fields_list):
+        res = super().default_get(fields_list)
+        is_model = self.env["ir.sequence"].sudo()
+        fields_with_seq = [
+            s.code[len(PROJECT_FIELDS_SEQUENCE_CODE_BASE) :]
+            for s in is_model.search(
+                [
+                    ("code", "ilike", PROJECT_FIELDS_SEQUENCE_CODE_BASE + "%"),
+                ]
+            )
+        ]
```

Comment:
This will improve performance, readability and accuracy:

```suggestion
        fields_with_seq = {
            seq.code[len(PROJECT_FIELDS_SEQUENCE_CODE_BASE) :]
            for seq in is_model.search(
                [
                    ("code", "=like", PROJECT_FIELDS_SEQUENCE_CODE_BASE + "%"),
                ]
            )
        }
```